pyvrp/educate/LocalSearch.py

```diff
@@ -124,7 +124,9 @@ def run(
 
             new_individual = self.intensify(individual, penalty_manager)
 
-            if new_individual.cost() < individual.cost():
+            if new_individual.cost(penalty_manager) < individual.cost(
+                penalty_manager
+            ):
```

Comment:
To avoid these awkward `black`-related expansions I typically assign to a variable first:
```python
new_individual = ...
new_cost = new_individual.cost(penalty_manager)

if new_cost < individual.cost(penalty_manager):
   ...
```

Reply:
This will be fixed with #191 anyway so I'm not changing it now.